log/model_valuer.go

```diff
@@ -93,6 +94,10 @@ func ByteBase64(b []byte) Valuer {
 }
 
 func Stringer(s fmt.Stringer) Valuer {
+	if reflect.ValueOf(s).Kind() == reflect.Pointer && reflect.ValueOf(s).IsNil() {
+		return &any{nil}
+	}
```

Comment:
I'm surprised checking `s == nil` doesn't work. 

Can we get the reflect value only once? 

```go
if v := reflect.ValueOf(s); v.Kind() == reflect.Pointer && v.IsNil() {
	return &any{nil}
}
```

Reply:
Yep same on surprise `s == nil` doesn't work.

And updated to get value of reflect once.